pyopenms_viz/_core.py

```
@@ -14,6 +14,10 @@
 from pandas.util._decorators import Appender
 import re
 
+import matplotlib.pyplot as plt
+from math import ceil
+import numpy as np
```

Ruff (0.8.2)

Redefinition of unused `np` from line 5

Remove definition: `np`

(F811)